scripts/compile_messages.sh

```diff
@@ -20,7 +20,7 @@ set -e
 cd static
 npm list @formatjs/cli || npm install formatjs
 
-# for LOCALE in de en es fr hi it ja ko pt-BR ru zh-CN;
+# for LOCALE in de en es fr hi it ja ko ru;
```

Comment:
you can delete the commented out line -- it should have been a TODO to add support for those languages (pybabel relies on local Locale files with underscore separators, and I couldn't figure out how to get around that).

Reply:
Done